ot/datasets.py

```diff
@@ -40,6 +40,34 @@ def get_1D_gauss(n, m, sigma):
     return make_1D_gauss(n, m, sigma)
 
 
+def make_1D_samples_gauss(n, m, sigma, random_state=None):
```

Comment:
this function is really doing 

https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html

I don't think POT needs such a function.

my 2c

Reply:
agreed